tests/features/rhpam/rhpam-kieserver.feature

```diff
@@ -1168,3 +1168,34 @@ Feature: RHPAM KIE Server configuration tests
     Then container log should contain -Dorg.jbpm.ejb.timer.tx=true
     And container log should contain -Dorg.jbpm.ejb.timer.local.cache=false
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
```

Comment:
both scenarios for rhpam and rhdm looks exactly the same, there is a kieserver-common feature for shared scenarios.

Reply:
Ok